Documentation/Contributing/ci-configuration.md

```diff
@@ -0,0 +1,13 @@
+---
+title: CI Configuration
+---
+
```

Comment:
An explanation of the purpose of this doc would be helpful, for example to say that these secrets are necessary for the CI to run against a fork.